js/src/launchAsync.ts

```diff
@@ -209,8 +219,14 @@ export function launchAsync(token: string, subdomain: string, content: Content,
                             pause: () => {
                                 iframe.contentWindow!.postMessage(JSON.stringify({ messageType: 'InstrumentationCommand', messageValue: pauseMessageValue }), '*');
                             },
+                            pauseIfWasPlaying: () => {
+                                iframe.contentWindow!.postMessage(JSON.stringify({ messageType: 'InstrumentationCommand', messageValue: pauseIfWasPlayingMessageValue }), '*');
```

Comment:
you only need to change the message value in existing Play/Pause operations.

We only want to expose Play/Pause functions, but behind the scenes, have them call PlayIfWasPaused/PauseIfWasPlaying

Reply:
Customers may still want to use play and pause though?